README.md

```diff
@@ -192,6 +192,35 @@ const { data, total } = await audits.getAudits({
 });
 ```
 
+### Entitlements Client
+
+#### Initializing the client
+```javascript
+
+const { EntitlementsClient } = require('@frontegg/client');
+
+// initialize the FronteggContext
+FronteggContext.init(
+  {
+    FRONTEGG_CLIENT_ID: '<YOUR_CLIENT_ID>',
+    FRONTEGG_API_KEY: '<YOUR_API_KEY>',
+  },
+  {
+    accessTokensOptions,
+  },
+);
+
+// initialize entitlements client
+const client = await EntitlementsClient.init(/* */);
+await client.ready();
+```
```

Comment:
Need to add usage examples. Can be done later as well.

Reply:
sure thing, for now no API is added actually :) this PR was only about periodical state update, but new functions are coming with next JIRA stories